### ISSUE TITLE
preference "area with data" don't works anymore as expected

### ISSUE THREAD

**Issue by dex2000** [JOSM/Mapillary#49]
Since last update of mapillary-plugin mapillary loads the whole affected sentences.

Before update it only loads parts of tracks near the downloaded osm-data.

Is there any way to get the old behaviour?

**Comment by floscher** [comment]
**TL;DR:** Check the checkbox "Cut off sequences at download bounds" in the preferences (see screenshot below) and you'll get the old behaviour back.

That change was intentional and affects all download modes. Previously you only got all images inside the area (bei it the visible area or the area with downloaded OSM-data).
Now you get all sequences with at least one image inside the area.

There are two reasons for that:
* These images that are now shown and previously not shown are downloaded anyways. The Mapillary API even returns more images, which are still discarded (it returns all sequences, whose bounding box intersects the requested bounding box).
* Several people asked, why they needed to download more data in order to follow a sequence. The previous behaviour was a bit misleading in that it looked as if a sequence ended, but in reality it continued outside the downloaded bounds. Especially when a sequence went outside the downloaded area and re-entered it somewhere else.

So this is now the default behaviour. But if you want, you can still get the old behaviour back by going into the preferences (shortcut `F12`) and check the last checkbox "Cut off sequences at download bounds".

![grafik](https://cloud.githubusercontent.com/assets/3904348/25231162/16a86262-25d7-11e7-82d5-5c0a12114db7.png)

May I ask what's the reason, you want the old behaviour back? Is it that the plugin gets too slow, because it loads too many images? Or do you maybe want to save internet bandwidth (which isn't the case)?

**Comment by dex2000** [comment]
works fine, thx
